jest.config.js

```diff
@@ -3,4 +3,6 @@ module.exports = {
   testEnvironment: 'node',
   testMatch: ['**/?(*.)+(spec|integration).ts'],
   testPathIgnorePatterns: ['/dist/'],
+  // TODO: remove the following if you don't have any localstack integration tests
```

Comment:
```suggestion
  // TODO: remove the following if you don't have any setup you would like to do before your tests run. Ex. setting a different endpoint for localstack services for integration tests
```

Reply:
i was under the impression that if localstack was in use, an endpoint had to be defined - e.g. `http://localstack:4566`. is that not the case?

Reply:
My comment is more to the point that this file is not just for setting up env vars for localstack, you can pretty much do anything in here like, for example, create a tmp directory for all tests to output something to.